app/controllers/lives_controller.rb

```diff
@@ -22,23 +26,25 @@ def create
   end
 
   def edit
-    @life = Life.find(params[:id])
   end
 
   def update
-    @life = Life.find(params[:id])
     @life.update(params[:life])
     redirect_to life_path(@life)
   end
 
   def destroy
```

Comment:
Does this need authorisation also?

Reply:
It is authorised im det_life, disregard previous comment

Reply:
think it should all be covered off in the private set_life method